### DISCUSSION TITLE
Which network has priority when using multiple network interfaces?

### DISCUSSION THREAD

**Discussion by jmm-slamcore** [iRobotEducation/create3_docs#247] [Questions]
It might be useful to add more documentation (or point users to existing documentation) around using multiple network interfaces with the create 3 (Ethernet over USB-C and Wi-Fi).

If the create 3 is:
- Connected to e.g. a raspberry pi using ethernet-over-usb 
- Also connected to Wi-Fi
 
is there a default network that is used to communicate with the Pi (if the Pi is also connected to Wi-Fi)? I imagine this is down to the chosen DDS and how it is configured on each side. In our use case, we would like for communication with the Pi to happen over USB preferably if available and Wi-Fi to be used to communicate with a laptop/wireless devices. 

Related questions:
- https://answers.ros.org/question/384008/ros2-prioritize-wifi-over-ethernet-by-default-whitelist-interface/

**Comment by alsora** [comment]
Hi @jmm-slamcore!
Yes, I agree that this is an important use-case that we are not documenting.

Unfortunately we don't have ready-to-use XML configurations for this scenario.
We will investigate the issue and expand the documentation.

**Reply by jmm-slamcore** [reply]
Thanks @alsora!

**Reply by rickarmstrong** [reply]
@alsora I'm also very interested in documentation of this setup.
Thanks,
Rick
